src/components/dataset/query/sidebar/QuerySidebarPanel.jsx

```diff
@@ -29,18 +20,28 @@ export class QuerySidebarPanel extends React.PureComponent {
     dataset: PropTypes.object,
     dispatch: PropTypes.func.isRequired,
     filterData: PropTypes.object,
+    results: PropTypes.number,
+    polling: PropTypes.bool,
     selected: PropTypes.string,
   };
 
   render() {
-    const { filterData, selected } = this.props;
+    const { classes, filterData, selected, results, polling } = this.props;
     const listTables = _.keys(filterData).map(table => {
       return <AppliedFilterList table={table} selected={selected} />;
     });
 
+    const resultsLabel = results == 1 ? 'Result' : 'Results';
+
     return (
       <Card>
-        <List subheader={<ListSubheader component="div">Properties</ListSubheader>}>
```

Comment:
Will users still need this text somewhere?

Reply:
I think we decided yesterday in the design meeting that it wasn't necessary, especially since it looks weird when there are no filters applied. We can add it back in somewhere else if Jerome wants it.

Reply:
I think we decided yesterday in the design meeting that it wasn't necessary, especially since it looks weird when there are no filters applied. We can add it back in somewhere else if Jerome wants it.